cmd/libs/go2idl/conversion-gen/generators/conversion.go

```diff
@@ -122,6 +133,16 @@ func existingConversionFunctions(context *generator.Context) conversions {
 	return preexisting
 }
 
+func isAPIPackage(path string) bool {
+	apiPackages := []string{"k8s.io/kubernetes/pkg/api/", "k8s.io/kubernetes/pkg/apis/"}
```

Comment:
This would make it impossible to use against another repo-- can we come up with a different rule?

It's ok to hard code k8s.io/kubernetes/pkg/api/ since that's an exception.

 Maybe contains "apis" and has a types.go file?